asyncdb/drivers/sqlite.py

```diff
@@ -221,7 +221,7 @@ async def execute_many(self, sentence: Union[str, list], *args) -> Optional[Any]
             if result:
                 await self._connection.commit()
         except Exception as err:
-            error = "Error on Execute Many: {err}"
+            error = f"Error on Execute Many: {err}"
```

Comment:
it should be fine as it is since it is done this way every where, also for the other drivers, this would be a general improvement which should be addressed in a dedicated commit if needed.